manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml

```diff
@@ -0,0 +1,44 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Waterfox.Waterfox
+PackageVersion: G6.0.3
+PackageLocale: en-US
+Publisher: WaterfoxLimited
+PublisherUrl: https://www.waterfox.net
+PublisherSupportUrl: https://github.com/WaterfoxCo/Waterfox/issues
+PrivacyUrl: https://www.waterfox.net/docs/policies/privacy
+Author: Alexandros Kontos, Adam Wood
+PackageName: Waterfox
+PackageUrl: https://www.waterfox.net
+License: Mozilla Public License
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0
+Copyright: © 2023 Waterfox Ltd, all rights reserved.
+CopyrightUrl: http://www.mozilla.org/foundation/licensing.html
+ShortDescription: FOSS fork of Mozilla Firefox focused on performance and privacy.
```

Comment:
Probably best to update this to:
> Waterfox is an independent, open-source web browser built for speed, privacy, and user customization.

Reply:
How about performance, privacy and customization? I don’t see much reason to change the rest

Reply:
The rest is important as Waterfox Limited is no longer the legal entity Waterfox belongs to, BrowserWorks is.

Reply:
The short description does not reference any legal entities.